flexget/plugins/input/parameterize.py

```diff
@@ -0,0 +1,62 @@
+from __future__ import unicode_literals, division, absolute_import
+from builtins import *  # noqa pylint: disable=unused-import, redefined-builtin
+
+import logging
+
+from flexget import plugin
+from flexget.event import event
+from flexget.utils.template import render_from_entry, RenderError
+
+log = logging.getLogger('parameterize')
+
+
+class Parameterize(object):
+
+    schema = {
+        'type': 'object',
+        'properties': {
+            # TODO: This should really be input plugin validation, but some format validators don't deal well with Jinja
+            'plugin': {'type': 'object'},
+            'using': {'$ref': '/schema/plugins?phase=input'},
+        },
+    }
+
+    def on_task_input(self, task, config):
+        for param_input_name, param_input_config in config['using'].items():
+            param_input = plugin.get_plugin_by_name(param_input_name)
+            method = param_input.phase_handlers['input']
+            try:
+                param_entries = method(task, param_input_config)
+            except plugin.PluginError as e:
+                log.warning('Error during input plugin %s: %s' % (param_input_name, e))
+                continue
+            for param_entry in param_entries:
+                for subj_input_name, subj_input_config in config['plugin'].items():
+                    subj_input = plugin.get_plugin_by_name(subj_input_name)
+                    method = subj_input.phase_handlers['input']
+                    subj_input_config = _parameterize(subj_input_config, param_entry)
+                    try:
+                        result = method(task, subj_input_config)
+                    except plugin.PluginError as e:
```

Comment:
i get why you're catching `PluginError` but I wonder if this could have adverse side effects since the expectation is to abort the task

Reply:
It's there right now because I copied it from somewhere else. 😉 But I'm not sure what the expectation should be actually. It's not guaranteed that if one of the sets of parameters fail that they all will. Would we still want it to try the others?